mitodl/mit-open
# Homepage News and Events section

### What are the relevant tickets?

[Homepage News & Events #4254](https://github.com/mitodl/hq/issues/4254)

### Description (What does it do?)

Adds the "Browse by Topics" section to the homepage:

[Desktop designs](https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=2606-57499&m=dev)
[Mobile designs](https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=241-4258&m=dev)


### Screenshots (if appropriate):

Desktop:
<img width="800" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/6abc2193-4e88-4f91-a42f-52ad68dd0fb4">

Mobile:
<img width="300" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/2c093ca6-ad80-40f2-86cd-50bfb6915189">



### How can this be tested?

Run the front end:

```
yarn watch:rc
```

- Load the homepage at http://localhost:8062/

- Ensure the "Experience MIT" section renders according to designs for desktop and mobile.

- Ensure images, titles and dates render.

- Ensure links navigate to sources.
